### ISSUE TITLE
I cannot drag and drop an image/file into a markdown cell

### ISSUE THREAD

**Issue by acrodrig** [HappenApps/Quiver#300]
It would be nice, just as I can drag/drop an image/file into a text cell, to be able to do it in a MarkDown cell. My current process for achieving this is quite laborious:
1. Create a Text Cell at the bottom of the note
2. Drag/Drop Image into it
3. Inspect element to figure out image internal URL
4. Create MarkDown image markup using said URL
5. Delete Text Cell at the bottom

I would love if dropping the image/file automatically created the markdown markup.

Cheers!

**Comment by johnpneumann** [comment]
:+1:

**Comment by aeroechelon** [comment]
I do agree that having this feature would be awesome – however in the meanwhile you can optimize this workflow:
1. Create a Text Cell below current Markdown Cell.
2. Drag and drop tmage into it.
3. Convert to Markdown cell.
4. Place cursor at beginning of cell.
5. Press 'backspace' to merge to upper Markdown cell.

**Comment by joachimd** [comment]
I totally agree that this ability should get added as I mainly work with MarkDown as my default cell type.  The only thing I can add is that I find it easier to insert an image in a MarkDown cell by doing Format->Insert Image from the menubar and then drag and drop the image into the file finder window that comes up.  That will create the MarkDown URL for you.

**Comment by egel** [comment]
I think this feature should also support for drag and drop multiple images/files not only one.
